src/core/a-entity.js

```diff
@@ -539,7 +539,7 @@ var proto = Object.create(ANode.prototype, {
    */
   setAttribute: {
     value: function (attr, value, componentPropValue) {
-      var isDebugMode = this.sceneEl && this.sceneEl.getAttribute('debug');
+      var isDebugMode = this.sceneEl && this.sceneEl.hasAttribute('debug');
```

Comment:
Steps look like:

- https://github.com/aframevr/aframe/blob/master/src/core/schema.js#L124 - Pass in the default value from the prop definition to to parse function.
- https://github.com/aframevr/aframe/blob/master/src/core/propertyTypes.js#L67 - Treat empty string as wanting the default value.